docs/Project.toml

```diff
@@ -3,5 +3,6 @@ CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
 DocumenterTools = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
 DynamicalBilliards = "4986ee89-4ee5-5cef-b6b8-e49ba721d7a5"
+GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
```

Comment:
Why are we adding this to the doc build? CairoMakie can also visualize the GUIs. You just can't click on them, but you anyways can't do this when compiling the docs in CI.

Reply:
Rushed finalising my commit and left in from testing whoops